examples/custom-matrix-format/CMakeLists.txt

```diff
@@ -18,13 +18,5 @@ set(CMAKE_CUDA_STANDARD_REQUIRED ON)
 add_executable(custom-matrix-format custom-matrix-format.cpp stencil_kernel.cu)
```

Comment:
need to set the arch here

Reply:
No, that's handled by the CMAKE_CUDA_ARCHITECTURES variable

Reply:
but is it set here?
It can not use the CMAKE_CUDA_ARCHITECTURES from Ginkgo, right?

Reply:
I think we support standalone examples as well, in which case it will not be set (we don't inherit it from the `find_package(Ginkgo)` I think? -- and probably should not)